src/Core/NuGet/Core.NuGet.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'Common.props'))\Common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>net45</TargetFrameworks>
+    <TargetFrameworks>netstandard1.3</TargetFrameworks>
```

Comment:
it is just the "last" target so the packing of the NuGet will happen after Base SDK were compiled for all projects.